app/uk/gov/hmrc/transitmovementsrouter/config/AppConfig.scala

```diff
@@ -46,4 +48,23 @@ class AppConfig @Inject() (config: Configuration, servicesConfig: CTCServicesCon
   lazy val objectStoreUrl: String =
     config.get[String]("microservice.services.object-store.sdes-host")
 
+  // SDES configuration
+  lazy val useProxy: Boolean = config.get[Boolean]("sdes.use-proxy") // set true for sdes proxy and false for sdes stub
+  lazy val service: String   = if (useProxy) "secure-data-exchange-proxy" else "sdes-stub"
```

Comment:
No need for this -- if you're just switching between the two just put the configuration in `secure-data-exchange-proxy`.